tests/console/system_prepare.pm

```diff
@@ -121,6 +122,22 @@ sub run {
 
     # stop and disable PackageKit
     quit_packagekit;
+    ########
+    record_info "agama pscc de-register";
+    zypper_call('in suseconnect-ng');
+    assert_script_run("suseconnect -d");
+    record_info "agama pscc register";
+    my $regcode = get_var('SCC_REGCODE');
+    my $regurl = get_var('SCC_URL');
+    assert_script_run("suseconnect -r $regcode --url $regurl");
+    zypper_call("up", timeout => 300);
```

Comment:
`dup` should be better to use, as we are doing major changed and changing repository.